analysis/lang/pl/stempel/fuzz.go

```diff
@@ -0,0 +1,35 @@
+//  Copyright (c) 2018 Couchbase, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 		http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build gofuzz
+// +build gofuzz
+
+package stempel
+
+var fuzzTrie Trie
+
+func init() {
+	var err error
+	fuzzTrie, err = Open("pl/stemmer_20000.tbl")
```

Comment:
Would you throw in a comment here on what the contents of the file are?

Reply:
It's prefix tree (trie) data, please see https://lucene.apache.org/core/6_6_0//analyzers-stempel/index.html?org/egothor/stemmer/class-use/Trie.html and https://github.com/blevesearch/bleve/issues/275 (@mschoch contribution)